pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -689,6 +731,17 @@ func publishAppInstanceStatus(ctx *zedmanagerContext,
 	key := status.Key()
 	log.Tracef("publishAppInstanceStatus(%s)", key)
 	pub := ctx.pubAppInstanceStatus
+	if ctx.hvTypeKube {
+		sub := ctx.subENClusterAppStatus
+		st, _ := sub.Get(key)
+		if st != nil {
+			clusterStatus := st.(types.ENClusterAppStatus)
+			if !clusterStatus.ScheduledOnThisNode {
```

Comment:
No publishing seems odd.

I can understand that we want the controller to ignore Info messages (and perhaps also Metrics messages) for app instances which are not scheduled on this node, but it would make more sense to either put that check in the controller (which might require a some new fields in the Info and Metrics protobuf messages) or put the code in zedagent to exclude sending Info and Metrics for such an app instance. But publish here for other agents to see a Status for each Config.